appinventor/components/src/com/google/appinventor/components/runtime/TextBoxBase.java

```diff
@@ -557,6 +522,43 @@ public String PropertyURI() {
     return propertyUri;
   }
 
+  private String conceptUri = "";
+  /**
+   * ConceptURI getter method.
+   *
+   * @return  concept uri
+   */
+  @SimpleProperty(
+          category = PropertyCategory.LINKED_DATA,
```

Comment:
Continuation indents are +4 from the previous scope.

Reply:
I reduced the indent. Sorry, I'm unsure about the AppInventor coding style. Feel free to update if it's not fixed yet.